src/Servers.jl

```diff
@@ -1,101 +1,74 @@
+"""
+The `HTTP.Servers` module provides server-side http functionality in pure Julia.
+
+The main entry point is `HTTP.listen(handler, host, port; kw...)` which takes a `handler` argument (see `?HTTP.Handlers`),
+a `host` and `port` and optional keyword arguments. For full details, see `?HTTP.listen`.
+"""
 module Servers
 
+export listen, handle, Handler, RequestHandler, StreamHandler
+
 using ..IOExtras
 using ..Streams
 using ..Messages
 using ..Parsers
 using ..ConnectionPool
-using ..Sockets
-import ..@info, ..@warn, ..@error, ..@debug, ..@debugshow, ..DEBUG_LEVEL, ..stdout
-using MbedTLS: SSLConfig, SSLContext, setup!, associate!, hostname!, handshake!
```

Comment:
I like the change of using fully qualified names to call e.g. `MbedTLS.setup.,
But, I don't think saying `MbedTLS.SSLConfig` or `MbedTLS.SSLContext` adds anything.
I think we should we should `MbedTLS: SSLConfig, SSLContext`.
The problem with just `using MbedTLS` is that it brings in a bunch of short generic names like `digest` and `encrypt`: https://github.com/JuliaWeb/MbedTLS.jl/blob/master/src/MbedTLS.jl#L21